docs/api/core/useStateObservable.md

```diff
@@ -0,0 +1,72 @@
+---
+title: useStateObservable(observable)
+sidebar_label: useStateObservable()
+---
+
+Gets the latest value from an observable returned by `state`.
+
+```ts
+function useStateObservable<T>(
+  observable: StateObservable<T>,
+): Exclude<T, typeof SUSPENSE>
+```
+
+#### Arguments
+
+- `observable`: The `StateObservable` to get the value from.
+
+#### Returns
+
+The latest emitted value of the Observable.
+
+If the Observable hasn't emitted a value yet, it will leverage React Suspense
+while it's waiting for the first value.
+
+### Examples
+
+```tsx
+import { scan } from "rxjs/operators"
+import { state, useStateObservable } from "@react-rxjs/core"
+
+const counter$ = state(clicks$.pipe(startWith(0)), 0)
+
+function CounterDisplay() {
+  const counter = useStateObservable(counter$)
+
+  return <div>{counter}</div>
+}
+```
+
+:::note
+It's important to note that the `StateObservable` must be created outside the React Component, otherwise it could run into an infinite loop.
```

Comment:
```suggestion
It's important to note that the `StateObservable` must be created outside the React render function, otherwise it could run into an infinite loop.
```